quiz.js

```diff
@@ -7,26 +7,22 @@ let quizTime = 0;
 let numWrong = 0;
 let numCorrect = 0;
 
-for (i = 0; i < 3; i++) {
+for (let i = 0; i < questions.length; i++) {
     let difficulty = 1;
-    tstart = new Date().getTime();
```

Comment:
If you don't re-assign `tstart` here you can just check once at the end of the script for how long it took.